src/components/pages/login/LoginForm.js

```diff
@@ -1,18 +1,33 @@
 import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom';
 
 export default function LoginForm() {
 
     // state
     const [newName, setNewName] = useState("");
+        // is authorized to access the order page
+        const [goToOrderPage, setGoToOrderPage] = useState(false);
```

Comment:
```suggestion
        const [isAuthorisedUser, setIsAuthorisedUser] = useState(false);
```


isQuelqueChose ou hasQuelqueChose ou shouldQuelqueChose